lib/storage/leveldb.go

```diff
@@ -100,8 +102,8 @@ func (st *LevelDBBackend) Has(k string) (bool, error) {
 func (st *LevelDBBackend) GetRaw(k string) (b []byte, err error) {
 	var exists bool
 	if exists, err = st.Has(k); !exists || err != nil {
-		if !exists {
-			err = fmt.Errorf("key, '%s' does not exists", k)
+		if !exists || err == leveldb.ErrNotFound {
+			err = sebakerror.ErrorStorageRecordDoesNotExist
```

Comment:
I didn't understand this code because I didn't know golang initialize variables in return type.
I don't decide my stands for this feature, but I want to talk if we use this feature heavily or not.

Reply:
I personally find named return very confusing, but it's quite heavily used through Sebak. It would be nice to have this discussion, yes.

Reply:
um.. 
according to this feature, should we define existing error like this??
for example; 

''' func (st *LevelDBBackend) Discard() error {
	ts, ok := st.core.(*leveldb.Transaction)
	if !ok {
		return errors.New("this is not *leveldb.Transaction")
	}

	ts.Discard()
	return nil
}

func (st *LevelDBBackend) Commit() error {
	ts, ok := st.core.(*leveldb.Transaction)
	if !ok {
		return errors.New("this is not *leveldb.Transaction")
	}

	return ts.Commit()
} ''' 

are have same error message.

Reply:
@cmcm2222 your code block is broken.

Reply:
@Geod24 @outsideris IMHO, the named return is outside of this PR.  :-) for discussing it, let make a new issue or do something else. :-)

@cmcm2222 I am not sure but that we should return a custom error from that methods. I think that it also is outside of this PR :-) Let we discuss it! :-)  But in my opinion, that errors must not be returned because of this cause by type miscasting. When we meet that errors, It is a bug.

Reply:
@outsideris yep I've changed that. thanks. 
@anarcher Thank you for kind explanation. so helpful.